src/framework/theme/components/checkbox/checkbox.component.ts

```diff
@@ -91,27 +211,56 @@ export class NbCheckboxComponent implements ControlValueAccessor {
   }
```

Comment:
let's also remove all named inputs and use `_` as a prefix for private vars, as  `_disabled`.